.travis.yml

```diff
@@ -1,12 +1,16 @@
 language: python
 python:
 - '2.7'
-- '2.6'
```

Comment:
I was actually a little unsure about what version to use here. Since we're using tox, the python version set here just determines which python interpreter is used for installing and running tox, so there's no point to having more than one version (putting multiple versions here would lead to redundant tests, right?).

I guess 3.4 is more appropriate now. I have vague superstitions about python 2.7 support still being more robust, but I don't think that's actually based on anything. I'll switch it to 3.4.